tests/main/security-dev-input-event-denied/task.yaml

```diff
@@ -22,7 +22,7 @@ restore: |
     rm -f call.error
 
 execute: |
-    if [ ! -d /dev/input/by-path/ ]; then
+    if [ ! -d /dev/input/by-path/ ] || [ "$(find /dev/input/by-path -name '*-event-kbd' | wc -l)" -eq 0 ]; then
```

Comment:
I think the whole test can be replaced with:
```
if [ -z "$(find /dev/input/by-path -name '*-event-kbd')" ]; then 
    ...
fi
```
So if /dev/input/by-path/ does not exist the output is empty, and if no *-event-kbd matches are found, the output is empty too.